axlearn/cloud/gcp/measurement.py

```diff
@@ -37,10 +41,22 @@ class Config(measurement.Recorder.Config):
         Attributes:
             upload_dir: Directory to store metrics for the monitor.
             upload_interval: Time interval (seconds) for monitoring uploads.
+            include_step_deviation: Whether to include step deviation
+              metrics in the upload.
+            step_deviation_interval_seconds: Time interval (seconds) for step
+              deviation metrics uploads.
+            enable_gcp_goodput_metrics: Whether to push Goodput metrics to
+              Google Cloud Monitoring.
+            enable_gcp_step_deviation_metrics: Whether to push step deviation
+              metrics to Google Cloud Monitoring.
         """
 
         upload_dir: Required[str] = REQUIRED
         upload_interval: Required[int] = REQUIRED
+        include_step_deviation: bool = True  # Default to True
+        step_deviation_interval_seconds: int = 30  # Default to 30 seconds
+        enable_gcp_goodput_metrics: bool = True  # Default to True
```

Comment:
Whether it is true or false, the goodput value will still show up on tensorboard, is it correct?